pkgs/development/compilers/zulu/common.nix

```diff
@@ -141,6 +141,18 @@ let
         fi
       '';
 
+    doInstallCheck = true;
+    installCheckPhase = ''
+      echo "Checking 'java -version' output contains ${dist.jdkVersion}, 'OpenJDK', and 'Zulu'"
+      output=`$out/bin/java -version 2>&1`
+      # Our Zulu 23 package is a special case (and should be deleted as it is End-of-Life)
```

Comment:
Is there a PR for that or shall we make one?

Reply:
PR #421587, but see also PR #403247